src/components/BrandConfigProvider/Styles.ts

```diff
@@ -0,0 +1,18 @@
+import { TextStyles } from '../Text/TextStyles';
+import { Theme } from './Theme';
+
+interface Props {
+  theme?: Theme;
+}
+
+export class Styles {
+  Text: TextStyles;
+
+  private _theme: Theme;
+
+  constructor({ theme }: Props = {}) {
+    this._theme = theme || new Theme();
+
+    this.Text = new TextStyles(this._theme);
+  }
+}
```

Comment:
This is where we can add style for any other components.  They will get a reference to the them injected into them.